styles/index.css

```diff
@@ -0,0 +1,374 @@
+* {
+	box-sizing: border-box;
+	padding: 0;
+	margin: 0;
+	font-family: 'Montserrat', Arial, Helvetica, sans-serif;
+}
+
+html {
+	font-size: 62.5%;
+}
+
+a {
+	text-decoration: none;
+}
+
+.row {
+	width: 100%;
+}
+
+.container {
+	max-width: 1150px;
+	margin: 0 auto;
+	display: flex;
+	padding: 0 2em;
+}
+.cta-button {
+	background-color: #25f18b;
+	border: none;
+	border-radius: 10px;
+	color: #fff;
+	cursor: pointer;
+	font-size: 1.8rem;
+	padding: 1em 2em;
+	transition: 0.3s ease background-color, 0.3s ease box-shadow,
+		0.3s ease transform;
+}
+
+.cta-button:hover {
+	background-color: #6cfeb5;
+	box-shadow: 5px 5px 20px rgba(0, 0, 0, 0.3);
+	transform: scale(1.1);
+}
+
+/* HEADER */
+
+.header {
+	background: radial-gradient(rgb(194, 74, 126), rgb(105, 86, 235));
+	color: #fff;
+	padding: 4.4em 0;
+}
+
+.header__navbar {
+	justify-content: space-between;
+	padding-right: 2em;
+}
+
+.header__logo img {
+	height: 3em;
+}
+
+.header__nav ul {
+	column-gap: 2.5em;
+	display: flex;
+	font-size: 1.5rem;
+	list-style-type: none;
+}
+
+.header__nav a {
+	color: #fff;
+	font-weight: 500;
+	transition: 0.3s ease color;
+}
+
+.header__nav a:hover {
+	color: #58d4f7;
+}
+
+.header__container {
+	margin-top: 5em;
+	justify-content: space-between;
+	padding-bottom: 7em;
+}
+
+.header__container .column {
```

Comment:
Jeśli potrzebujesz ograniczyć "style" (przyporządkować dla konkretnego przypadku) to robisz Modyfikator tj. np. `.column--header` i te dwie klasy dodajesz do elementu np. `<div class="column column--header">`